root/static/scripts/common/constants.js

```diff
@@ -149,6 +149,7 @@ export const FAVICON_CLASSES = {
   'rockipedia.no': 'rockipedia',
   'rolldabeats.com': 'rolldabeats',
   'runeberg.org': 'runeberg',
+  'saisaibatake.ame-zaiku.com': 'gakki',
```

Comment:
```suggestion
  'saisaibatake.ame-zaiku.com/gakki/': 'gakki',
```

Reply:
I left it without because we don't have /wiki for generasia, but maybe we should just have that too.

Reply:
I suggested it because you were using it for [URL specialization](https://github.com/metabrainz/musicbrainz-server/commit/02c8301d150a37bbc9ef84c97ea8f86de386b93d#diff-c7e5aef95d55e2d6e1d96236e898f6701a422ce0ced4a2b4c701a97d3b4ff639R71) and [URL cleanup](https://github.com/metabrainz/musicbrainz-server/commit/02c8301d150a37bbc9ef84c97ea8f86de386b93d#diff-a808bbcd17cb3e956e9ac0fce554d75f6e9db02e020bc03ca543c198ac8f8e06R1356) already, and I saw it was used for [dancedb favicon class](https://github.com/metabrainz/musicbrainz-server/blob/v-2020-10-19/root/static/scripts/common/constants.js#L163) too.

> we don't have /wiki for generasia, but maybe we should just have that too.

:+1:

Reply:
Ok, pushed that change.